src/Core/src/Platform/Windows/WrapperView.cs

```diff
@@ -120,6 +120,7 @@ void UpdateClip()
 			var pathGeometry = compositor.CreatePathGeometry(path);
 			var geometricClip = compositor.CreateGeometricClip(pathGeometry);
 
+			geometricClip.Offset = new Vector2(-Child.ActualOffset.X, -Child.ActualOffset.Y);
```

Comment:
Could include a comment explaining why apply these values here?

`// Align clip geometry with the child's visual offset to prevent masking misalignment blablabla`